## A non-industrialized pig gut microbiomes reveals an untapped source of peptide antibiotics

**Folder and File Structure**
- pipeline: metagenomics and genomics pipeline, including assembly, binning, geneset construction, taxonomic and functional annotation, etc.
- scripts: some R, Python, Shell, Perl scripts used in this study
- FigureX: the scripts and necessary files for each figure

**The database and software used in this study as follow:**
- fastp (v0.23.2)
- SPAdes (v3.13.0)
- Bowtie2 (v2.4.4)
- MEGAHIT (v1.2.9)
- Prodigal (v2.6.1)
- MMseqs2 (v14-7e284)
- DIAMOND (v2.1.8.162)
- MetaBAT2 (v2.15)
- Vamb (v4.1.3)
- BWA (v0.7.17-r1188)
- dRep (v3.4.2)
- CheckM2 (v1.0.1)
- INFERNAL (v1.1.4)
- tRNAscan-SE (v2.0.12)
- GTDB-Tk (v2.3.2)
- PhyloPhlAn (v1.0)
- iTOL (v6.3.2)
- Minimap2 (v2.26-r1175)
- APEX (v1.1)
- CoverM (v0.7.0)
- AntiSMASH (v6.1.1)
- BiG-SCAPE (v1.1.5)
- BiG-SLiCE (v1.1.0)
- libsqlite (v3.43.2)
- R environment (v4.3.1)
- vegan (v2.6-10)
- metafor (v3.4)
- ggplot2 (v3.5.1)
- Python environment (v3.11.6)
- pandas (v2.1.4)
- numpy (v1.26.2)
- biopython (v1.83)
- scipy (v1.11.4)
- modlamp (v4.3.2)
